academic-services/graph/network-similarity-usql-gettopentities.md

```diff
@@ -0,0 +1,60 @@
+---
+title: NetworkSimilarity.GetTopEntities Function
+description: NetworkSimilarity.GetTopEntities Function
+services: microsoft-academic-services
+ms.topic: extra
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# NetworkSimilarity.GetTopEntities Function
+
+Namespace: AcademicGraph.NetworkSimilarity
+
+### Returns entities with top similarity scores related to an entity.
+
+## GetTopEntities(BaseDir, EntityType, Sense, EntityId, MaxCount, MinScore)
```

Comment:
>(BaseDir, EntityType, Sense, EntityId, MaxCount, MinScore [](start = 17, length = 57)

Same as other comment

Reply:
The Python methods parameters are camel cased. USQL functions parameters are capitalized.

---
In reply to: [501883030](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/183#discussion_r501883030) [](ancestors = 501883030)